### PR TITLE
Provide info for inconsistent relations

### PR DESCRIPTION
**Pull request by Vegter**

### CODE REVIEW

File: src/gobupload/storage/relate.py [hunk 1]
```diff
@@ -879,15 +882,25 @@ def _check_relate_update(new_values, src_field_name, src_identification):
...
             {new_values}
         ) new_values
     """
-    control_data = _get_data(control_query)
-    inconsistencies = next(control_data)['count']
 
-    if inconsistencies > 0:
-        logger.error(f"{src_field_name}, {inconsistencies} inconsistencies found")
+    errors = 0
+    error_msg = f"Conflicting {src_field_name} relations"
```
Comment (JKoetsier) [inline src/gobupload/storage/relate.py]: Maybe we should provide a better explanation in this error message of what a "conflict" exactly is?

Comment (Vegter) [reply src/gobupload/storage/relate.py]: Well, I could not find any better solution then just showing the conflict. You then clearly see the current relation and the conflicting one. The same bronwaarde maps onto two different id's, which is a conflict.

Comment (Vegter) [reply src/gobupload/storage/relate.py]: logger.error(error_msg, {
                'id': error_msg,
                'data': data
            })

Comment (Vegter) [reply src/gobupload/storage/relate.py]: where data is:
{src_identification},
            {src_field_name}_updated AS relation,
            src_matchcolumn          AS conflits_with

Comment (Vegter) [reply src/gobupload/storage/relate.py]: I tested this locally and found the explanation clear enough. It gives enough clues to solve or to investigate the conflict

Comment (Vegter) [reply src/gobupload/storage/relate.py]: Question posted in GOB channel....

**Pull request merged**